crates/ironclaw_webui/frontend/src/styles/app.css

```
@@ -394,9 +398,57 @@ input:focus, select:focus, textarea:focus {
+   Another intentional motion-policy exception: while the agent is working, a
+   light chases the N's spine (`near-chase`) and the glyph does a quick spin +
+   heartbeat (`near-pulse`). Both use a class selector + !important so they
+   outrank the universal `animation: none` rule, and both are suppressed under
+   prefers-reduced-motion below. */
+.near-process-icon {
+  color: var(--near-blue);
+  overflow: visible;
+  transform-box: fill-box;
+  transform-origin: center;
+}
+.near-process-icon .near-base { fill: currentColor; }
```

Stylelint (17.14.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)